### PR TITLE
Make createAccount page compliant with requirements of current dev hub prototype

### PR DESCRIPTION
**Pull request by ch-rfishwick**
__Short description outlining key changes/additions__
Made createAccount page compliant with requirements of current dev hub prototype

__JIRA Ticket Number__
FA-128

### Type of change

*  Bug fix
* [ ] New feature
* [ ] Breaking change

### Pull request checklist

* [ ] I have added unit tests for new code that I have added
* [ ] I have added/updated functional tests where appropriate
* The code follows our [coding standards](https://github.com/companieshouse/styleguides/blob/master/java.md)

__An exhaustive list of peer review checks can be read [here](https://github.com/companieshouse/styleguides/blob/master/java_review.md#developer-actions-prior-to-code-commitreview-started)__

### CODE REVIEW

File: src/main/resources/templates/dev-hub/createAccount.html [hunk 1]
```diff
@@ -8,9 +8,9 @@
 </head>
 <body>
 <div layout:fragment="content">
-  <h1 class="heading-large" th:utext="${templateTitle}">Set up a CH Account</h1>
+  <h1 class="govuk-heading-xl" th:utext="${templateTitle}">Set up a CH Account</h1>
   <p>To be able to explore and perform tests with the Companies House API, you
-    need to register a <a href="/developer/signin">user account</a> with Companies House.</p>
+    need to register a <a href="http://www.example.com">user account</a> with Companies House.</p>
```
Comment (CH-JonathanBlake) [inline src/main/resources/templates/dev-hub/createAccount.html]: In all other places we are using https, should we break from this standard?

File: src/main/resources/templates/dev-hub/createAccount.html [hunk 2]
```diff
@@ -8,9 +8,9 @@
 </head>
 <body>
 <div layout:fragment="content">
-  <h1 class="heading-large" th:utext="${templateTitle}">Set up a CH Account</h1>
+  <h1 class="govuk-heading-xl" th:utext="${templateTitle}">Set up a CH Account</h1>
   <p>To be able to explore and perform tests with the Companies House API, you
```
Comment (ch-dclarke) [inline src/main/resources/templates/dev-hub/createAccount.html]: p tags require a govuk styling class

File: src/main/resources/templates/dev-hub/createAccount.html [hunk 3]
```diff
@@ -8,9 +8,9 @@
...
-    need to register a <a href="/developer/signin">user account</a> with Companies House.</p>
+    need to register a <a href="http://www.example.com">user account</a> with Companies House.</p>
 </div>
```
Comment (ch-dclarke) [inline src/main/resources/templates/dev-hub/createAccount.html]: Are these indents correct?

File: src/main/resources/templates/dev-hub/createAccount.html (hunk 1 above)
Comment (ch-dclarke) [inline src/main/resources/templates/dev-hub/createAccount.html]: Please style the link appropriately

**Pull request merged**